src/core/Database.cpp

```diff
@@ -276,14 +276,16 @@ bool Database::saveAs(const QString& filePath, QString* error, bool atomic, bool
     return ok;
 }
 
-bool Database::performSave(const QString& filePath, QString* error, bool atomic, bool backup)
+bool Database::performSave(const QString& filePath, SaveFlags flags, QString* error)
 {
 #if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
     QFileInfo info(filePath);
     auto createTime = info.exists() ? info.birthTime() : QDateTime::currentDateTime();
 #endif
 
-    if (atomic) {
+    bool backup = flags.testFlag(Backup);
```

Comment:
I think that variable is quite redundant and I would prefer the actual flag test in-place.

Reply:
Overcome by other changes